gninasrc/lib/model.h

```diff
@@ -255,10 +255,16 @@ struct ligand : public flexible_body, atom_range {
 
 struct residue : public main_branch {
     residue() {
-    } //serialization
+    } // for serialization
     residue(const main_branch& m)
         : main_branch(m) {
     }
+

```

Comment:
Serialisation seemed to be missing here (compared to similar classes), but I might be wrong!

Reply:
Removed from this PR and opened a different PR

Reply:
Moved to a different PR